docs/querying.md

```diff
@@ -135,6 +135,8 @@ $like: '%hat',         // LIKE '%hat'
 $notLike: '%hat'       // NOT LIKE '%hat'
 $iLike: '%hat'         // ILIKE '%hat' (case insensitive) (PG only)
 $notILike: '%hat'      // NOT ILIKE '%hat'  (PG only)
+$regexp: '^[h|a|t]'    // REGEXP '^[h|a|t]' (MySQL only)
+$notRegexp: '^[h|a|t]' // NOT REGEXP '^[h|a|t]' (MySQL only)
```

Comment:
Go over my most recent commit and see if this is still something you would like done.